third_party/ruby_root.bzl

```
@@ -0,0 +1,20 @@
+# This rule is used to get the RUBY_ROOT environment variable from the
+# user's local environment and make it available to the build system.
+def _ruby_root_impl(repository_ctx):
+  # Retrieve the RUBY_ROOT environment variable
+  ruby_root = repository_ctx.os.environ.get("RUBY_ROOT")
+
+  if not ruby_root:
+    fail("RUBY_ROOT environment variable is not set.")
+
+  # Create a file to store the RUBY_ROOT value
+  content = 'RUBY_ROOT = "{}"\n'.format(ruby_root)
```

Do we have access to f strings by any chance?

```suggestion
  content = f"RUBY_ROOT = \"{ruby_root}\"\n"
```